atm.js

```diff
@@ -1,21 +1,96 @@
+    var checkingTotalAfterWithdrawal = (parseFloat($("div.balance").eq(0).html().substr(1))) + checkingAccountUserInput;
+    $("div.balance").eq(0).html("$" + checkingTotalAfterWithdrawal.toFixed(2));
+    $("input").eq(0).val("");
+    $(".account").eq(0).removeClass("zero");
+  }
+});
+$("input").eq(3).on("keypress", function(event) {
+  if(event.which == 13) {
+    event.preventDefault();
+    var savingsAccountUserInput = Number(parseFloat($("input").eq(3).val()));
+    var savingsTotal = (parseFloat($("div.balance").eq(1).html().substr(1))) + savingsAccountUserInput;
+    $("div.balance").eq(1).html("$" + savingsTotal.toFixed(2));
+    $("input").eq(3).val("");
+    $(".account").eq(1).removeClass("zero");
+  }
+});
+function depositIntoChecking(event) {
+  event.preventDefault();
+  var checkingAccountUserInput = Number(parseFloat($("input").eq(0).val()));
+  var checkingTotalAfterWithdrawal = (parseFloat($("div.balance").eq(0).html().substr(1))) + checkingAccountUserInput;
+  $("div.balance").eq(0).html("$" + checkingTotalAfterWithdrawal.toFixed(2));
+  $("input").eq(0).val("");
+  $(".account").eq(0).removeClass("zero");
 }
-
-function checkingWithdrawal(amount){
-
+function depositIntoSavings(event) {
+  event.preventDefault();
+  var savingsAccountUserInput = Number(parseFloat($("input").eq(3).val()));
+  var savingsTotal = (parseFloat($("div.balance").eq(1).html().substr(1))) + savingsAccountUserInput;
+  $("div.balance").eq(1).html("$" + savingsTotal.toFixed(2));
+  $("input").eq(3).val("");
+  $(".account").eq(1).removeClass("zero");
 }
-
-function savingsDeposit(amount){
-
+function withdrawFromChecking(event) {
+  event.preventDefault();
+  var checkingAccountUserWithdrawal = Number(parseFloat($("input").eq(0).val()));
+  var checkingTotalAfterWithdrawal = (parseFloat($("div.balance").eq(0).html().substr(1))) - checkingAccountUserWithdrawal;
+  var savingsTotal = (parseFloat($("div.balance").eq(1).html().substr(1)));
+  if(checkingTotalAfterWithdrawal + savingsTotal < 0) {
+    alert("Sorry! You have insufficient Funds!");
+  }
+  else if(checkingTotalAfterWithdrawal > 0){
+    $("div.balance").eq(0).html("$" + checkingTotalAfterWithdrawal.toFixed(2));
+    $("input").eq(0).val("");
+  }
+  else{
+    var currentAmountInChecking = (parseFloat($("div.balance").eq(0).html().substr(1)));
+    var totalAvailableFunds = currentAmountInChecking + savingsTotal;
+    console.log(totalAvailableFunds);
+    var amountToWithdrawFromSavings = checkingAccountUserWithdrawal - currentAmountInChecking;
+    savingsTotal = savingsTotal - amountToWithdrawFromSavings;
+    $("div.balance").eq(0).html("$0.00");
+    $(".account").eq(0).addClass("zero");
+    if(savingsTotal === 0) {
+      $(".account").eq(1).addClass("zero");
+    }
+    $("div.balance").eq(1).html("$" + savingsTotal.toFixed(2));
+    $("input").eq(0).val("");
+  }
 }
-
-function savingsWithdrawal(amount){
-
+function withdrawFromSavings(event) {
```

Comment:
This function is also almost entirely duplicated compared to the function above, only the two accounts are switched. Wonder if you could re-write as one function, that takes two inputs, it might look like this:
```js
function withdrawFunds(amount, primaryAccount, overdraftAccount) {
  // your code here
}

withdrawFunds(100, checkingAccount, savingsAccount);
```